pages/api/create_class_teacher.js

```diff
@@ -0,0 +1,18 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
```

Comment:
You need to use the prisma singleton object that oliver created to avoid multiple connections to the DB.
`import prisma from '../../prisma/prisma';`